src/pages/Fox/components/RFOXSection.tsx

```diff
@@ -145,11 +144,6 @@ export const RFOXSection = () => {
     return matchingAccountId
   }, [accountIdsByAccountNumberAndChainId, assetAccountNumber, stakingAssetId])
 
-  const stakingAssetAccountAddress = useMemo(
```

Comment:
Goes from account address for RFOX query hooks to CAIP across the board for sanity (since we needed to have a amtching AccountId for multi-account, that is a more boilerplaty diff, but much cleaner than do an ugly `toAccountId()`)